theme/theme.ts

```diff
@@ -48,6 +48,21 @@ const IronFishTheme: DeepPartial<ChakraTheme> = {
       },
     }),
   },
+  layerStyles: {
+    card: {
+      bg: NAMED_COLORS.WHITE,
+      boxShadow: `0.25rem 0.25rem 0 -0.063rem ${NAMED_COLORS.WHITE}, 0.25rem 0.25rem ${NAMED_COLORS.LIGHT_GREY}`,
+      border: '0.063rem solid',
+      borderColor: NAMED_COLORS.LIGHT_GREY,
+      borderRadius: '0.125rem',
+      m: '0.2rem',
+      '.chakra-ui-dark &': {
```

Comment:
This done because for layerStyles color mode [doesn't work](https://github.com/chakra-ui/chakra-ui/issues/3451) for some reason